src/context/PreFetchData.tsx

```diff
@@ -16,6 +18,9 @@ interface PreFetchData {
 const PreFetchDataContext = createContext<PreFetchData | null>(null);
 
 const PreFetchDataProvider = ({ children }: BaseComponentProps) => {
+    // TODO (context) create a local storage context provider
+    useLocalStorage(LocalStorageKeys.CONNECTOR_TAG_SELECTOR, ':dev');
```

Comment:
Have the pre fetcher set the local storage for now. I think we should probably end up creating a local storage context to handle this.